### PR TITLE
ART-3107 Change operator channel names to stable (4.10)

### PR DESCRIPTION
**Pull request by thegreyd**
https://issues.redhat.com/browse/ART-3107

Test run:
./doozer --assembly=test -g openshift-4.10 olm-bundle:rebase-and-build --force sriov-network-operator-container-v4.10.0-202109150958.p0.git.5e1261d.assembly.stream

Changes visible at
- https://pkgs.devel.redhat.com/cgit/containers/sriov-network-operator-bundle/tree/metadata/annotations.yaml?h=rhaos-4.10-rhel-8
- https://pkgs.devel.redhat.com/cgit/containers/sriov-network-operator-bundle/tree/Dockerfile?h=rhaos-4.10-rhel-8

### CODE REVIEW

File: doozerlib/olm/bundle.py [hunk 1]
```diff
@@ -560,9 +560,17 @@ def redhat_delivery_tags(self):
 
     @property
     def operator_framework_tags(self):
+        version = '{MAJOR}.{MINOR}'.format(**self.runtime.group_config.vars)
+        override_channel = self.channel
+        override_default = self.channel
+        # wip see: issues.redhat.com/browse/ART-3107
+        if version == '4.10':
```
Comment (sosiouxme) [inline doozerlib/olm/bundle.py:567]: could this be a group.yml setting? I prefer not to have things this specific in code.

File: doozerlib/olm/bundle.py [hunk 2]
```diff
@@ -560,9 +560,17 @@ def redhat_delivery_tags(self):
...
+            override_channel = f'{self.channel},stable'
```
Comment (sosiouxme) [inline doozerlib/olm/bundle.py:568]: if the channel is already "stable" could this deduplicate?

**Review by sosiouxme — COMMENTED** [review]
it would work but...

File: doozerlib/olm/bundle.py [hunk 3]
```diff
@@ -560,9 +560,18 @@ def redhat_delivery_tags(self):
 
     @property
     def operator_framework_tags(self):
+        override_channel = self.channel
+        override_default = self.channel
+        stable_channel = 'stable'
+
+        # see: issues.redhat.com/browse/ART-3107
+        if self.runtime.group_config.operator_channel_stable:
+            override_channel = ','.join([self.channel, stable_channel])
```
Comment (sosiouxme) [inline doozerlib/olm/bundle.py:569]:
```suggestion
            override_channel = ','.join({self.channel, stable_channel})
```
... would ensure that they're unique, right?
i want the owners to switch their operator channels to `stable` in the source at least at next branch cut if not before.

Comment (thiagoalessio) [reply doozerlib/olm/bundle.py:569]: +1

**Review by thiagoalessio — APPROVED** [review]
/lgtm

**Pull request merged**